### ISSUE TITLE
Track ECN usage

### ISSUE THREAD

**Issue by ihlar** [EricssonResearch/spindump#51]
It would be useful to track and log ECN flags.
00 – Non ECN-Capable Transport, Non-ECT
10 – ECN Capable Transport, ECT(0)
01 – ECN Capable Transport, ECT(1)
11 – Congestion Encountered, CE.

Observing a CE flag should generate an event.

**Comment by jariarkko** [comment]
Yes, that would be a good idea! Should be relatively simply to do that... (speaking as someone who is not an ECN expert...)

**Comment by ihlar** [comment]
It's simple, I'll take a stab at it :)

**Comment by gloinul** [comment]
It is easy as long as you are on a Linux or Mac system which has some API surface to read them.

**Comment by ihlar** [comment]
Actually in this case we don't need the API as we are capturing raw and parsing raw packets.

**Comment by britram** [comment]
@gloinul spindump observes packets though, so the api surface doesn’t matter...
